examples/dataservice-read/example.cpp

```diff
@@ -34,12 +34,71 @@
 namespace {
```

Comment:
Maybe it is time to change to proper google style -> https://google.github.io/styleguide/cppguide.html#Constant_Names